src/librbd/ObjectMap.cc

```diff
@@ -163,8 +163,14 @@ void ObjectMap<I>::close(Context *on_finish) {
     return;
   }
 
-  auto req = object_map::UnlockRequest<I>::create(m_image_ctx, ctx);
-  req->send();
+  ctx = new LambdaContext([this, on_finish](int r) {
```

Comment:
@dillaman `create_context_callback` above (line 159) is not used now, the valgrind reports about a memory leak [1]

[1] http://qa-proxy.ceph.com/teuthology/trociny-2020-06-19_18:13:08-rbd-wip-mgolub-testing-1-distro-basic-smithi/5164909/teuthology.log

Reply:
Fixed, thanks.